app/views/sessions/new.html.erb

```diff
@@ -1,20 +1,39 @@
-<div class='sign-up-in-tile'>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>ACEBOOK by D-Railed</title>
+</head>
+<body>
+  <h1>ACEBOOK</h1>
+  
+  <div class='container'>
+    <div class='row'>
+      <div class='col-md-2'></div>
+        <div class='col-md-4'>
+          <ul clas='ul'>
+            <a class='a_signin' href='<%= new_session_path %>'>Sign in</a>
+            <a href='<%= root_path %>'> / Sign up</a>
```

Comment:
this has the same functionality as the previous code: <%= link_to "Sign Up", root_path %> ??